### ISSUE TITLE
Error using create_labeled_video- scorer was not used

### ISSUE THREAD

**Issue by ghost** [DeepLabCut/DeepLabCut#249]
Hi, i am using deeplabcut on jupyter notebook, ubuntu 16.04

**Describe the problem**
I have labeled and trained a network successfully, analyzed the video using the trained network but when using the function "create_labeled_video" i get the following error

"Starting %  /home/orel/panasonicDemo/none-orel-2019-03-14/videos ['/home/orel/panasonicDemo/none-orel-2019-03-14/videos/secondVideo.mp4']
Loading  /home/orel/panasonicDemo/none-orel-2019-03-14/videos/secondVideo.mp4 and data.
The video was not analyzed with this scorer: DeepCut_resnet50_noneMar14shuffle1_150000
No other scorers were found, please use the function 'analyze_videos' first."

However, I did use the 'analyze_video" function on the same video, for which i got the following message:
"The videos are analyzed. Now your research can truly start! 
 You can create labeled videos with 'create_labeled_video'.
If the tracking is not satisfactory for some videos, consider expanding the training set. You can use the function 'extract_outlier_frames' to extract any outlier frames!"

would appreciate you assistance

The config.yaml file for the project is the following:
# Project definitions (do not edit)
Task: none
scorer: orel
date: Mar14

# Project path (change when moving around)
project_path: /home/orel/panasonicDemo/none-orel-2019-03-14

# Annotation data set configuration (and individual video cropping parameters)
video_sets:
  /home/orel/panasonicDemo/none-orel-2019-03-14/videos/secondVideo.MP4:
    crop: 0, 1920, 0, 1080
bodyparts:
- Head1
- Head2
- Head3
- Back1
- Back2
- Tailbase
- ForepawR
- HindpawR
- ForepawL
- HindpawL
start: 0
stop: 1
numframes2pick: 500

# Plotting configuration
pcutoff: 0.1
dotsize: 12
alphavalue: 0.7
colormap: hsv

# Training,Evaluation and Analysis configuration
TrainingFraction:
- 0.95
iteration: 0
resnet: 50
snapshotindex: -1
batch_size: 4

# Cropping Parameters (for analysis and outlier frame detection)
cropping: false
#if cropping is true for analysis, then set the values here:
x1: 0
x2: 640
y1: 277
y2: 624

# Refinement configuration (parameters from annotation dataset configuration also relevant in this stage)
corner2move2:
- 50
- 50
move2corner: true

**Comment by MMathisLab** [comment]
Was there an .h5 file created next to the video you input? 
[I.e. in the folder where your video is stored?]

If it doesn't show a progress bar of analysis (please see cell [10] here: https://github.com/AlexEMG/DeepLabCut/blob/master/examples/Demo_labeledexample_Openfield.ipynb)  it means its likely not analyzed (because it was not found, see below)

**Comment by AlexEMG** [comment]
It is possible that your video is not found, e.g. by default only avi files in a folder are considered. Thus you might want to change the videotype in the call. Check out the following examples:

    If you want to analyze only 1 video
    >>> deeplabcut.analyze_videos('/analysis/project/reaching-task/config.yaml',['/analysis/project/videos/reachingvideo1.avi'])   
    If you want to analyze all videos of type avi in a folder:
    >>> deeplabcut.analyze_videos('/analysis/project/reaching-task/config.yaml',['/analysis/project/videos'],videotype='.avi')
    --------
    If you want to analyze multiple videos
    >>> deeplabcut.analyze_videos('/analysis/project/reaching-task/config.yaml',['/analysis/project/videos/reachingvideo1.avi','/analysis/project/videos/reachingvideo2.avi'])
    --------
    If you want to analyze multiple videos with shuffle = 2
    >>> deeplabcut.analyze_videos('/analysis/project/reaching-task/config.yaml',['/analysis/project/videos/reachingvideo1.avi','/analysis/project/videos/reachingvideo2.avi'], shuffle=2)
    --------
    If you want to analyze multiple videos with shuffle = 2 and save results as an additional csv file too
    >>> deeplabcut.analyze_videos('/analysis/project/reaching-task/config.yaml',['/analysis/project/videos/reachingvideo1.avi','/analysis/project/videos/reachingvideo2.avi'], shuffle=2,save_as_csv=True)

Can you check if it is loaded correctly?

**Comment by ghost** [comment]
Thank you! I used the example and found the problems. it didn't find the file as you said.
maybe you should add an error message when it doesn't
the problem was the videotype, and a small syntex issue
Thank you for you help

**Comment by MMathisLab** [comment]
Yes, video type is important. :) 

We don’t have an error message as when you start using this for lots of analysis, the idea is you can just place new videos into the folder (or sub folders), and you can set up automatic jobs to run the line deeplabcut.analyze_videos - if new videos are there, it runs! You don’t want to break this pipeline with a non-error message :)
